### ISSUE TITLE
Webpack error

### ISSUE THREAD

**Issue by lishine** [developit/unfetch#108]
When deploying on Zeit Now v2 , there is an error from webpack, that I found is due to this package
```
ModuleDependencyWarning: Critical dependency: the request of a dependency is an expression
                              at Compilation.reportDependencyErrorsAndWarnings (/tmp/2b34e4f6/node_modules/webpack/lib/Compilation.js:1359:23)
                              at Compilation.finish (/tmp/2b34e4f6/node_modules/webpack/lib/Compilation.js:1165:9)
                              at hooks.make.callAsync.err (/tmp/2b34e4f6/node_modules/webpack/lib/Compiler.js:544:17)
                              at _err0 (eval at create (/tmp/2b34e4f6/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:11:1)
                              at Promise.all.then (/tmp/2b34e4f6/node_modules/webpack/lib/DynamicEntryPlugin.js:74:20)
                              at <anonymous>
                              at process._tickCallback (internal/process/next_tick.js:188:7)
                              at CommonJsRequireContextDependency.getWarnings (/tmp/2b34e4f6/node_modules/webpack/lib/dependencies/ContextDependency.js:40:18)
                              at Compilation.reportDependencyErrorsAndWarnings (/tmp/2b34e4f6/node_modules/webpack/lib/Compilation.js:1354:24)
                              at Compilation.finish (/tmp/2b34e4f6/node_modules/webpack/lib/Compilation.js:1165:9)
                              at hooks.make.callAsync.err (/tmp/2b34e4f6/node_modules/webpack/lib/Compiler.js:544:17)
                              at _err0 (eval at create (/tmp/2b34e4f6/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:11:1)
                              at Promise.all.then (/tmp/2b34e4f6/node_modules/webpack/lib/DynamicEntryPlugin.js:74:20)
                              at <anonymous>
                              at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by developit** [comment]
@lishine how are you inferring that this is due to unfetch?

**Comment by lishine** [comment]
Hi :)
By way of elimination. I import file which  imports only isomorphic-unfetch and basically just doing fetch. And then I see this error. If I remove to this file, no error

**Comment by Skaronator** [comment]
I think the source of this warning comes from this file: https://github.com/developit/unfetch/blob/master/packages/isomorphic-unfetch/index.js

**Comment by Skaronator** [comment]
Looks like it's the fault of `node-fetch`, still getting these warnings after replacing isomorphic-unfetch by node-fetch:
```

ModuleDependencyWarning: Critical dependency: the request of a dependency is an expression
    at Compilation.reportDependencyErrorsAndWarnings (/tmp/24a8999e/node_modules/webpack/lib/Compilation.js:1361:23)
    at Compilation.finish (/tmp/24a8999e/node_modules/webpack/lib/Compilation.js:1167:9)
    at hooks.make.callAsync.err (/tmp/24a8999e/node_modules/webpack/lib/Compiler.js:622:17)
    at _err0 (eval at create (/tmp/24a8999e/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:11:1)
    at Promise.all.then (/tmp/24a8999e/node_modules/webpack/lib/DynamicEntryPlugin.js:74:20)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at CommonJsRequireContextDependency.getWarnings (/tmp/24a8999e/node_modules/webpack/lib/dependencies/ContextDependency.js:40:18)
    at Compilation.reportDependencyErrorsAndWarnings (/tmp/24a8999e/node_modules/webpack/lib/Compilation.js:1356:24)
    at Compilation.finish (/tmp/24a8999e/node_modules/webpack/lib/Compilation.js:1167:9)
    at hooks.make.callAsync.err (/tmp/24a8999e/node_modules/webpack/lib/Compiler.js:622:17)
    at _err0 (eval at create (/tmp/24a8999e/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:11:1)
    at Promise.all.then (/tmp/24a8999e/node_modules/webpack/lib/DynamicEntryPlugin.js:74:20)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by developit** [comment]
This is fixed by #133.
